### DISCUSSION TITLE
ilogtail  在k8s-v1.31.1版本上不能用吗？目测收集不到日志打到kafka上

### DISCUSSION THREAD

**Discussion by kubelsp** [alibaba/ilogtail#2127] [Help]
Before asking a question, please first consider:

- Searching [existing discussions](https://github.com/alibaba/ilogtail/discussions/)
- Searching [existing issues](https://github.com/alibaba/ilogtail/issues/)
- Searching [English Doc](https://github.com/alibaba/ilogtail/tree/main/docs/en)
- Searching Google

-------

提问之前，建议先在 iLogtail 社区寻找答案。

- 查看 [存在的 discussions](https://github.com/alibaba/ilogtail/discussions/)
- 查看 [存在的 issues](https://github.com/alibaba/ilogtail/issues/)
- 查看 [中文文档](https://ilogtail.gitbook.io/ilogtail-docs/about/readme)
- 查看 Google

如果仍然无法找到答案，我们建议您在[discussions](https://github.com/alibaba/ilogtail/discussions/)进行提问。

**Comment by Takuka0311** [comment]
k8s-v1.31.1应该是能够使用的，有更多信息吗？
